### PR TITLE
feat(conf): make LV_MEM_BUF_MAX_NUM configurable

### PR DESCRIPTION
**Pull request by xiaoxiang781216**
### Description of the feature or fix

Simple change

### Checkpoints
- [X] Follow the [styling guide](https://github.com/lvgl/lvgl/blob/master/docs/CODING_STYLE.md)
- [ ] Update [CHANGELOG.md](https://github.com/lvgl/lvgl/blob/master/docs/CHANGELOG.md)
- [ ] Update the documentation

### CODE REVIEW

File: lv_conf_template.h [hunk 1]
```diff
@@ -62,6 +62,9 @@
 #  define LV_MEM_CUSTOM_REALLOC realloc
 #endif     /*LV_MEM_CUSTOM*/
 
+/*Number of the memory buffer*/
```
Comment (kisvegabor) [inline lv_conf_template.h]: If it becomes public we should give a more detailed description.

For example:
```suggestion
/*Number of the intermediate memory buffer used during rendering and other internal processing mechanisms. 
 *You will see an error log message if there wasn't enough buffers. */
```

Comment (xiaoxiang781216) [reply lv_conf_template.h]: Done.

**Comment by kisvegabor** [comment]
CI reports a Kconfig error:
image

**Comment by xiaoxiang781216** [comment]
> CI reports a Kconfig error: image

Forget to add help keyword, should work now.

**Comment by kisvegabor** [comment]
Thanks!

**Pull request merged**